src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days >= 7) {
```

Comment:
For days: `3` and `7`, create separate values, cause now they are 'magic numbers'